lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -296,6 +308,24 @@ def _set_sgi_ip_addr(self, if_name: str):
         except subprocess.SubprocessError as e:
             self.logger.warning("Error while setting SGi IP: %s", e)
 
+    def _is_iface_ip_set(self, if_name, ip_addr):
+        mask = '/' in ip_addr
+
+        ifaddrs = netifaces.ifaddresses(if_name).get(netifaces.AF_INET, [])
+        for addr in ifaddrs:
+            parts = [addr['addr']]
+            if mask:
+                prefix = sum(bin(int(x)).count('1')
+                             for x in addr['netmask'].split('.'))
```

Comment:
lets avoid open coding the subnet calculation, can you use something like:
```
>>> import netaddr
>>> cidr = netaddr.IPNetwork('%s/%s' % ('1.1.1.0', '255.255.255.0'))
>>> print(cidr)
1.1.1.0/24
```

Reply:
Didn't notice that `netaddr` already a part of the project. I had no intention to add more dependencies, but once it here, made it way simpler.